samples/sandvich-ios/base.xcconfig

```diff
@@ -0,0 +1,8 @@
+#include "../../xcconfig/common.xcconfig"
+
+INFOPLIST_FILE = Example/Info.plist
+PRODUCT_BUNDLE_IDENTIFIER = org.mozilla.Example
```

Comment:
Maybe we should be more specific and go for `SandvichIos`

Reply:
Sure, it will avoid any naming conflicts.